OasisDEX/oasis-borrow
# Auto-take profit history

# [Auto-take profit history](https://app.shortcut.com/oazo-apps/story/5732/ui-auto-take-history-handling)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- added handling for auto-take profit history
  
## How to test 🧪
  <Please explain how to test your changes>

- add, update, remove auto-take profit triggers and check whether history is correct
- history of execution not yet available but should be correct as well
